Source/Core/Core/HW/WiimoteCommon/WiimoteHid.h

```diff
@@ -0,0 +1,38 @@
+// Copyright 2008 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include "Common/CommonTypes.h"
+
+// what is this ?
+#ifdef _MSC_VER
+#pragma warning(disable : 4200)
+#endif
+
+#pragma pack(push, 1)
+
+// Source: HID_010_SPC_PFL/1.0 (official HID specification)
+
+struct hid_packet
+{
+  u8 param : 4;
+  u8 type : 4;
+  u8 data[0];
+};
+
+#define HID_TYPE_HANDSHAKE 0
```

Comment:
Can these be constexpr?

Also, these should probably be in a namespace, otherwise you're polluting the global namespace.